Packages/io.chainsafe.web3-unity/Runtime/GUI/GuiInfoOverlay.cs

```diff
@@ -8,14 +8,18 @@ namespace ChainSafe.Gaming.GUI
     public class GuiInfoOverlay : MonoBehaviour
     {
         public TMP_Text Message;
+        public TMP_Text ToastMessage;
```

Comment:
Why not reuse 'Message'?

Reply:
because toast message has a different positioning (bottom) and is under a layout group that has a `MaxSize` component on it